app/public/covid.css

```diff
@@ -141,6 +147,9 @@ div.doubleblock {
   height: 90%;
   padding: 7px;
 }
+.hero {
+  max-width: 820px!important;
```

Comment:
820px is still too big, I would recommend 500px
![image](https://user-images.githubusercontent.com/8701973/146539452-80c51c16-e713-4bd1-8754-d33962004912.png)

I have run the test locally as we can not run them in GitHub actions given you don't have the keys for the test environments in you clone repo

Reply:
looks good can you please fix conflicts before we can merge